recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
     fn: cmocean-{{ version }}.tar.gz
     url: https://github.com/matplotlib/cmocean/archive/{{ version }}.tar.gz
-    sha256: 22e40329f5fd1c6d2136d7b32dfe988f3b8570a360b86bf9786efb2cb20c9aaa
+    sha256: 7f52753cd7cb2a8ccd532dadbce8db149088e27e3ad9b9f05ec9a30b6c12db4c
```

Comment:
Yes I'm on a mac. Is it not the same?

On Thu, Jul 21, 2016 at 11:53 AM, Filipe <notifications@github.com> wrote:

> In recipe/meta.yaml
> <https://github.com/conda-forge/cmocean-feedstock/pull/2#discussion_r71765271>
> :
>
> > @@ -7,7 +7,7 @@ package:
> >  source:
> >      fn: cmocean-{{ version }}.tar.gz
> >      url: https://github.com/matplotlib/cmocean/archive/{{ version }}.tar.gz
> > -    sha256: 22e40329f5fd1c6d2136d7b32dfe988f3b8570a360b86bf9786efb2cb20c9aaa
> > +    sha256: 7f52753cd7cb2a8ccd532dadbce8db149088e27e3ad9b9f05ec9a30b6c12db4c
>
> Are you on a Mac? I believe that computing sha256 is different and maybe
> @jjhelmus <https://github.com/jjhelmus> can you me remember the command.
>
> I just computed it on my Linux box and I got:
> ccc5aa3e277f7f163ac58a2670326f48310ec5154b1c85d4fb1069d623fdfa20
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/cmocean-feedstock/pull/2/files/80a767d09897aba71d984815767b9421485f62af#r71765271>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADU2BcBXjk2ttdaeRAmj0SfBwksnr6udks5qX8ApgaJpZM4JSFl_>
> .
>



-- 
Kristen M. Thyng
Assistant Research Professor
Department of Oceanography
Texas A&M University
Eller O&M 607
http://kristenthyng.com